bangazonreports/views/products/inexpensiveproducts.py

```diff
@@ -0,0 +1,38 @@
+import sqlite3
+from django.shortcuts import render
+from bangazonapi.models import Product
+from bangazonreports.views import Connection
+
+
+def inexpensive_list(request):
+    if request.method == 'GET':
+        with sqlite3.connect(Connection.db_path) as conn:
+            conn.row_factory = sqlite3.Row
+            db_cursor = conn.cursor()
+
+            db_cursor.execute("""
+                select
+                    p.id,
+                    p.name,
+                    p.description,
+                    p.price
+                from bangazonapi_product p
+                where p.price <= 999
```

Comment:
What if the price is 999.50?

Reply:
I had it <1000 but I am going based of your ticket description as 

"As a business aanalyst, I would like to see an HTML report showing all products priced at $999 or less"

But I can change if you want. Thank you

Reply:
Modified the logic to include price < 1000